### PR TITLE
Feature: ASB-1780 Register to Have Your Say Navigation

### PR DESCRIPTION
**Pull request by christopher-semmens**
### Task
https://pins-ds.atlassian.net/browse/ASB-1780?atlOrigin=eyJpIjoiNWVjNDQyMDIzZjkzNDU3MTg2YWZiMzdjMjdmZmMzYzUiLCJwIjoiaiJ9

### CODE REVIEW

**Review by Philip-Philippou1 — APPROVED** [review]

File: packages/forms-web-app/src/pages/projects/register/index/_utils/is-registration-open.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const dayjs = require('dayjs');
+const isSameOrAfter = require('dayjs/plugin/isSameOrAfter');
+
+dayjs.extend(isSameOrAfter);
+
+const isRegistrationOpen = (openDate, closeDate) => {
+	const dayToday = dayjs();
+	const registrationOpenDate = dayjs(openDate);
+	const registrationCloseDate = dayjs(closeDate);
```
Comment (emil-placheta) [inline packages/forms-web-app/src/pages/projects/register/index/_utils/is-registration-open.js:9]: shouldn't registration stay open until the close date 23:59:59?
eg: `const registrationCloseDate = dayjs(closeDate).endOf('day');`

I might be wrong, not sure how the registration should work, ignore if it's meant to close on the day

**Pull request merged**